### PR TITLE
US06: Add new ingcesarvelasquez.json file

### PR DESCRIPTION
**Pull request by ingcesarvelasquez**

### CODE REVIEW

File: ingcesarvelasquez.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+"name": "Cesar Andres Velasquez",
```
Comment (danielcb29) [inline ingcesarvelasquez.json]: Please change your name to lower case

**Comment by ingcesarvelasquez** [comment]
+1 to merge

**Comment by Fabian0007** [comment]
+1

**Comment by danielcb29** [comment]
+1 to merge

**Pull request merged**